src/gapstate.h

```diff
@@ -148,6 +148,7 @@ typedef struct GAPState {
     Obj  ErrorLVars0;       // the initial ErrorLVars value, i.e. for the lvars were the break occurred
     Obj  ErrorLVars;        // ErrorLVars as modified by DownEnv / UpEnv
     Int  ErrorLLevel;       // record where on the stack ErrorLVars is relative to the top, i.e. ErrorLVars0
+    void (*JumpToCatchFunc)(); // Error catching function for GAP shared library
```

Comment:
This `JumpToCatchFunc` is a bit mysterious, and of course currently never set. And this comment doesn't really help ("shared library? what shared library?")

Perhaps "Callback invoked by FuncJUMP_TO_CATCH" would be more helpful here, and then a somewhat longer explanation in "FuncJUMP_TO_CATCH", like:  "Optionally invoke a callback just before we jump. This is not used by GAP itself, but exists for use in code linking against GAP."